src/XAMLTools.Core/XAMLCombine/XAMLCombiner.cs

```diff
@@ -300,12 +369,26 @@ private static void ChangeNamespacePrefix(XmlElement element, string oldPrefix,
                         attr.Prefix = newPrefix;
                     }
 
-                    // Check {x:Type {x:Static in attributes values
-                    // TODO: Is any other???
-                    if ((attr.Value.Contains("{x:Type") || attr.Value.Contains("{x:Static"))
-                        && attr.Value.Contains(oldStringSpaced))
+                    if (attr.Value.Contains(oldStringSpaced))
                     {
-                        attr.Value = attr.Value.Replace(oldStringSpaced, newStringSpaced);
+                        // Check {x:Type {x:Static in attributes values
+                        // TODO: Is any other???
+                        if (attr.Value.Contains("{x:Type") || attr.Value.Contains("{x:Static"))
```

Comment:
what if someone, for some reason, does not use x: ? Should we check on :Type or "full namespace"?